Firmware/ControladorRobo/irarray.cpp

```diff
@@ -70,14 +72,57 @@ void IRArray::readSensors()
     for (int i = 0; i < NUMBER_OF_SENSORS; i++)
     {
         m_sensorRaw[i] = analogRead(m_sensorPin[i]);
-        byte sensorRawByte = m_sensorRaw[i]>>2;
-        if (m_calibrating)
```

Comment:
Maybe
Continuous calibration has it's issues, when we take the robot off the table, for instance.
We should discuss this in an issue, probably.